jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -1441,7 +1711,137 @@ function Get-ProtocolTypeAssociation {
     return $manifestList
 }
 ##### END MIT License #####
+$Script:ErrorMessage = $null
+function Error-Map {
```

Comment:
Error-Map is the cause for failing tests, we should be using an approved verb to identify that this function does: https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.4

`Write-AdmuErrorMessage` would be my suggestion